performance/PerformanceBaselines.json

```diff
@@ -2,77 +2,90 @@
     "PerformanceRuntime": {
         "InputBinding": {
             "TestName": "GetProductsTest",
-            "RunTimeAverageSec": 2.081
+            "RunTimeAverageSec": 2.081,
+            "RunTimeThresholdSec": 0.208
```

Comment:
Could you instead have a `RunTimeAverageMinSec` and `RunTimeAverageMaxSec`? That way comparing the numbers would be super simple